Client/Frontend/Browser/BrowserViewController/BrowserViewController+KeyCommands.swift

```diff
@@ -9,27 +9,27 @@ import UIKit
 extension BrowserViewController {
     fileprivate typealias shortcuts = String.KeyboardShortcuts
 
-    @objc private func openSettingsKeyCommand() {
+    @objc internal func openSettingsKeyCommand() {
```

Comment:
nitpick: I think we can remove internal entirely and it will be `internal` automatically for us.